src/test/groovy/graphql/execution/AsyncExecutionStrategyTest.groovy

```diff
@@ -117,6 +117,7 @@ abstract class AsyncExecutionStrategyTest extends Specification {
                 .newParameters()
                 .executionStepInfo(typeInfo)
                 .fields(mergedSelectionSet(['hello': mergedField([Field.newField('hello').build()]), 'hello2': mergedField([Field.newField('hello2').build()])]))
+                .nonNullFieldValidator(new NonNullableFieldValidator(executionContext))
```

Comment:
The rest of the changes are test fixups where  they half create support objects with just enough brains to work